test/pipeline_test_runner.cc

```diff
@@ -328,6 +341,11 @@ TEST_CASE("PerPhaseTest") { // NOLINT
         handler.addObserved(*gs, "flatten-tree", [&]() { return resolvedTree.tree->toString(*gs); });
         handler.addObserved(*gs, "flatten-tree-raw", [&]() { return resolvedTree.tree->showRaw(*gs); });
 
+        // Don't run typecheck on RBI files.
```

Comment:
Apparently I'm the first to introduce rbi files into `test/testdata` that have `sig`s. Running typecheck on them causes Sorbet to complain that they aren't returning the correct data type.